### DISCUSSION TITLE
AWS Bucket Policy error

### DISCUSSION THREAD

**Discussion by ekramper** [bullhorn/career-portal#450] [Q&A]
Charles,

We have uploaded the files for a career portal. When I go to configure our bucket policy, I am getting an error message. 

![image](https://user-images.githubusercontent.com/86618060/145868995-03d36875-2a6a-476c-a430-bf0212382956.png)

What have a done wrong?

Eric

**Comment by charlesabarnes** [comment]
Hello, Do you have the error message or screen shot of the error on hand?

**Comment by ekramper** [comment]
***@***.***



***@***.***

**Reply by charlesabarnes** [reply]
hello, email replies do not include attachments

**Comment by ekramper** [comment]
![image](https://user-images.githubusercontent.com/86618060/146048086-5e03a71f-295f-41c4-8247-c5ccee17cdc9.png)

![image](https://user-images.githubusercontent.com/86618060/146048115-2b610d55-9d6b-4fed-a501-40dbb5221096.png)

**Reply by charlesabarnes** [reply]
After the bucket name you are supposed to have `/*` in it.

**Comment by ekramper** [comment]
Can you see it on Github. I just cut and pasted the screenshots

**Comment by ekramper** [comment]
Thank you!

**Comment by ekramper** [comment]
Everything is configured in AWS. When I click on the link

http://kalosolutionscareers.s3-website.us-east-2.amazonaws.com/

I am getting an error message.

**Reply by charlesabarnes** [reply]
Please contact Bullhorn Support, looks like there are some things that need to be configured.

**Comment by ekramper** [comment]
Ok.  Thanks
